app/src/test/java/org/oppia/android/app/story/StoryActivityLocalTest.kt

```diff
@@ -66,29 +83,53 @@ private const val TEST_STORY_ID = "GJ2rLXRKD5hw"
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(
   application = StoryActivityLocalTest.TestApplication::class,
-  qualifiers = "port-xxhdpi"
+  qualifiers = "sw600dp"
```

Comment:
Do we only want this test file for tablets?

Reply:
There was only one test case earlier which was independent of device thats why I made this change instead of introducing qualifier to each test.

Reply:
It seems reasonable then, we can maybe change this later when we add more tests here.

Reply:
Sounds good. Resolving now.